migrate.go

```diff
@@ -364,9 +364,11 @@ func PlanMigration(db *sql.DB, dialect string, m MigrationSource, dir MigrationD
 	if len(existingMigrations) > 0 {
 		result = append(result, ToCatchup(migrations, existingMigrations, record)...)
 	}
-
 	// Figure out which migrations to apply
-	toApply := ToApply(migrations, record.Id, dir)
+	toApply := ToApply(migrations, existingMigrations, dir)
+	if err != nil {
```

Comment:
Oops. At one point, I made `ToApply()` return an error, but changed my mind. I never took out the error check.